spec/components/provider_interface/application_card_component_spec.rb

```diff
@@ -146,9 +146,49 @@
       end
     end
 
+    context 'when undergraduate' do
+      let(:course_option) do
+        course_option_for_provider(
+          provider: current_provider,
+          course: create(
+            :course,
+            :teacher_degree_apprenticeship,
+            name: 'Alchemy',
+            provider: current_provider,
+            accredited_provider:,
+          ),
+          site: create(
+            :site,
+            code: 'L123',
+            name: 'Skywalker Training',
+            provider: current_provider,
+          ),
+          study_mode: 'part_time',
```

Comment:
Can any of this be removed? Do we need to set all these things to check the rendered text?

Reply:
Tbf, I was just trying to avoid to rewrite the whole spec and the way the lets are organised. So I just rewrote the let I needed

Reply:
I create a let course. Done